src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -109,9 +109,43 @@ internal override SyntaxNode ScopeDesignatorOpt
             get { return _scopeBinder.ScopeDesignator; }
         }
 
-        internal override uint RefEscapeScope => _refEscapeScope;
+        // From https://github.com/dotnet/csharplang/blob/main/proposals/low-level-struct-improvements.md:
+        //
+        // | Parameter or Local     | ref-safe-to-escape | safe-to-escape |
+        // |------------------------|--------------------|----------------|
+        // | Span<int> s            | current method     | calling method |
+        // | scoped Span<int> s     | current method     | current method |
+        // | ref Span<int> s        | calling method     | calling method |
+        // | scoped ref Span<int> s | current method     | calling method |
+        // | ref scoped Span<int> s | current method     | current method |
+
+        internal sealed override uint RefEscapeScope
+        {
+            get
+            {
+                if (!_scopeBinder.UseUpdatedEscapeRules ||
+                    _scope == DeclarationScope.Unscoped)
+                {
+                    return _refEscapeScope;
+                }
+                return Binder.TopLevelScope;
```

Comment:
>return Binder.TopLevelScope; [](http://example.com/codeflow?start=16&length=28)

It looks like the tale above talks about two different scopes, but we always return the same value. What am I missing?

Reply:
For all scoped values, we return _ref-safe-to-escape_ of current method; for unscoped values, we return _ref-safe-to-escape_ from the initializer.

Reply:
> for unscoped values, we return ref-safe-to-escape from the initializer.

Perhaps the table above should reflect that and the spec as well.